src/main/java/seedu/duke/data/exception/SystemException.java

```diff
@@ -39,6 +39,8 @@
 
         EXCEPTION_MISSING_KEYWORD("No search query input. Please enter a keyword for search results."),
 
+        EXCEPTION_NOTEBOOK_NOT_EXIST("This note does not exist in the notebook! "),
```

Comment:
No need for this exception since its the same as the COMMAND_UNSUCCESSFUL_MESSAGE in your deleteCommand. I have added my comment below for the function to return the boolean instead of throwing exception.